closed/get_j9_source.sh

```diff
@@ -51,10 +51,10 @@ declare -A git_urls
 declare -A shas
 
 git_urls[openj9]=https://github.com/eclipse/openj9
-branches[openj9]=master
+branches[openj9]=openj9-0.9.0-rc1
```

Comment:
Shouldn't these changes be made after branching?

Reply:
wasn't sure we were going to bother branching jdk9, since this is the last ever release....

Reply:
Interesting idea. @DanHeidinga @mstoodle any thoughts? It does make it clear that no further changes need to be made to the jdk9 extensions, even when OpenJ9 changes, such as for https://github.com/eclipse/openj9/pull/2189

Reply:
My preference is to have a `openj9-0.9.0` branch here so it's very clear what content was in the 0.9.0 release.  This makes it possible to rebuild the release if any changes are delivered to this repo after the 0.9 release.

It's also a consistency thing: all the extensions repos included in a release should have a branch for that release.  Naming the release branches consistently saves anyone from having to think about which branch to use.

Reply:
The clarity reasoning is the most compelling. For relatively little work, anyone looking back on this repo will be able to understand why this change was made if it's in a branch specific to 0.9.0 .

Reply:
Agree, i'll do the branch today